### PR TITLE
Bakta DM: Fix and add test file moving

### PR DESCRIPTION
**Pull request by bebatut**
FOR CONTRIBUTOR:
* [X] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [X] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc [hunk 1]
```diff
@@ -1,34 +1,5 @@
...
-V1.0_2022-10-12	7197299	1.4	/private/var/folders/7j/21czvpk170zcyjdbqrp0hmsh0000gn/T/tmpdn_di5v_/galaxy-dev/tool-data/bakta_database/7197299
-V5.1light_2024-01-19	11371737	1.7	/private/var/folders/7j/21czvpk170zcyjdbqrp0hmsh0000gn/T/tmpdn_di5v_/galaxy-dev/tool-data/bakta_database/11371737
-V5.0light_2023-06-08	8021027	1.8	/private/var/folders/7j/21czvpk170zcyjdbqrp0hmsh0000gn/T/tmpdn_di5v_/galaxy-dev/tool-data/bakta_database/8021027
-V1.0_2022-10-12	7197299	1.4	/private/var/folders/7j/21czvpk170zcyjdbqrp0hmsh0000gn/T/tmp7n51m5jb/galaxy-dev/tool-data/bakta_database/7197299
-V5.1light_2024-01-19	11371737	1.7	/private/var/folders/7j/21czvpk170zcyjdbqrp0hmsh0000gn/T/tmp7n51m5jb/galaxy-dev/tool-data/bakta_database/11371737
-V5.0light_2023-06-08	8021027	1.8	/private/var/folders/7j/21czvpk170zcyjdbqrp0hmsh0000gn/T/tmp7n51m5jb/galaxy-dev/tool-data/bakta_database/8021027
+V5.1light_2024-01-19	11371737	1.7	bakta-db
+V5.1light_2024-01-19	10522951	1.7	/private/var/folders/7j/21czvpk170zcyjdbqrp0hmsh0000gn/T/tmp71vmuq4u/galaxy-dev/tool-data/bakta_database/10522951
```
Comment (bernt-matthias) [inline data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc]:
```suggestion
```

To my knowledge: The test assumptions only test against the datamanager json file. During the test entries are added to the loc file, which might be useful to check for, but the tests do not use this info.

Comment (bebatut) [reply data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc]: Should we remove this file?

Comment (bernt-matthias) [reply data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc]: I think we need the file. But the content does not matter. So if you like you can empty it, but we can also merge as it is.

Comment (bebatut) [reply data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc]: It would be great to merge it so we can update the DMs on EU and FR

**Review by bernt-matthias — APPROVED** [review]

**Comment by bebatut** [comment]
Thanks @bernt-matthias and @bgruening

**Pull request merged**